Sources/StreamChat/Controllers/MessageController/MessageController.swift

```diff
@@ -80,7 +80,7 @@ public class ChatMessageController: DataController, DelegateCallable, DataStoreP
     public var listOrdering: MessageOrdering = .topToBottom {
         didSet {
             if state != .initialized {
-                _repliesObserver.reset()
```

Comment:
This is removed because we now `reset` before removing the store coordinator, right?